util/config/schema.py

```diff
@@ -1204,5 +1204,13 @@
             "description": "Automatically clean stale blobs leftover in the uploads storage folder from cancelled uploads",
             "x-example": False,
         },
+        "FEATURE_MIRROR_UNSIGNED_REGISTRIES": {
+            "type": "array",
+            "description": "Disable image signature verification when mirroring repositories for the list of registries",
+            "x-example": [
+                "registry.redhat.io",
+                "registry.access.redhat.com"
+            ],
+        }
```

Comment:
This should not be a registry wide flag. This should be a per-repo decision made by the repo owner, which means a UI option in the repo mirror configuration UI. The option should also be added to the API request with which a repo mirror configuration can be created.

Reply:
In order to have Skopeo copy from unsigned registries it requires changing the `policy.json` file in the quay deployment, which effects the entire registry. There does exist the `--insecure-policy` flag which may work with individual repositories. Let me look into possibly getting that working.

Reply:
The `--policy` switch uses a path to the file as an argument. Why not just add the code to change a path to the policy file depending on the switch in the UI? A much cleaner approach.

Reply:
That would be even better, an even cleaner design. But that again leaves us with a per-repo settings, not global wide settings. I don't think this is something that impacts the whole registry so that it requires a feature flag. We always mirror individual repositories anyway.

Reply:
I agree with that design. At the time it seemed using the `--insecure-policy` was less secure but it actually has the same effect as modifying the policy.json file. Let me look into having the configuration per-repo.